processing/src/main/java/org/apache/druid/segment/RowBasedStorageAdapter.java

```diff
@@ -167,6 +168,15 @@ public Metadata getMetadata()
     throw new UnsupportedOperationException("Cannot retrieve metadata");
   }
 
+  @Override
+  public List<String> getSortOrder()
+  {
+    // It's possibly incorrect in some cases for sort order to be SORTED_BY_TIME_ONLY here, but for historical reasons,
+    // we're keeping this in place for now. The handling of "interval" in "makeCursors", which has been in place for
+    // some time, suggests we think the data is always sorted by time.
```

Comment:
is this referring to the `RowWalker` with its `skipToDateTime`?

Reply:
Yes.